web/app/js/components/util/ApiHelpers.jsx

```diff
@@ -143,7 +143,7 @@ const ApiHelpers = (pathPrefix, defaultMetricsWindow = '1m') => {
     // Traffic Performance Summary. This retrieves (non-Prometheus) stats for the given resource.
     let resourceUrl = '/api/tps-reports?skip_stats=true&resource_type=' + type;
 
-    if (_isEmpty(namespace)) {
+    if (_isEmpty(namespace) || namespace === "_all") {
```

Comment:
Should this be really changed to `_all`? I see calls to `urlForResourceNoStats("all")` in `Tap.jsx`, `Top.jsx` and `TopRoutes.jsx`.
There are also calls to `urlForResource(..., "all")` above from `Namespace.jsx` and `ResourceDetail.jsx`

Reply:
`urlsForResourceNoStats()` and `urlsForResource()` both take two arguments, `type` and `namespace`.  

`Tap`, `Top` and `TopRoutes` pass in `"all"` for `type`. The sidebar redesign relates to the second `namespace` argument, where you can pass in `"_all"` to query the API for all namespaces. Agree that this is confusing, I'll rework this.